test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -43,4 +43,27 @@ private void assertAreUnique(Object... objects) {
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+    @Test
+    public void isAnyNull() {
+        // No arguments
+        assertFalse(Utils.isAnyNull());
```

Comment:
Good that you've used comments to separate the different types of test cases, since it improves readability of your code.